go.mod

```diff
@@ -37,6 +37,7 @@ require (
 	github.com/sarslanhan/cronmask v0.0.0-20230801193303-54e29300a091
 	github.com/sirupsen/logrus v1.9.3
 	github.com/sony/gobreaker v0.5.0
+	github.com/spaolacci/murmur3 v1.1.0
```

Comment:
please use github.com/cespare/xxhash/v2 instead. This one we already import, use and tested to be very good for sharding. From [benchmarks](https://xxhash.com/) it also looks like xxhash is 10x faster.